renderer/src/helpers/hooks/useTheme.ts

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from "react";
+import { useThemeStore } from "@/store/themeStore";
+import { shallow } from "zustand/shallow";
+
+const useTheme = () => {
+  const [isOSDarkTheme, setIsOSDarkTheme] = useState(true);
+
+  const [theme, setTheme] = useThemeStore(
+    (state) => [state.theme, state.setTheme],
+    shallow
+  );
+
+  function handleThemeChange(e) {
```

Comment:
Could you, please, overwrite this function?
Example` const handleThemeChange = (e) = >  e.matches ?  setIsOSDarkTheme(true) :  setIsOSDarkTheme(false);`
OR
You could test e.matches inside `setIsOSDarkTheme(Boolean(e.matches))` - it should be tested.